codebreaker-web-template/lose.html.erb

```diff
@@ -16,27 +16,27 @@
         <div class="col-12 col-md-6">
           <div class="row">
             <div class="col-md-auto">
-              <h1 class="text-white title inline-block">Codebreaker 2018</h1>
+              <h1 class="text-white title inline-block">Codebreaker 2019</h1>
             </div>
           </div>
           <div class="row">
             <div class="col button">
               <button type="button" class="btn btn-primary" disabled>
                 Level:
                 <span class="badge badge-light">
-                  Middle
+                  <%=level_to_word(@game.stats.difficulty)%>
```

Comment:
In common practice it looks like `<%= level_to_word(@game.stats.difficulty) %>` with whitespaces